spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe FoodService do 
+  it "gets data for food ingredient", :vcr do 
+    foods = FoodService.get_foods("sweet potatoes")
+    # require 'pry'; binding.pry 
+    expect(foods).to be_a(Array)
```

Comment:
I'd expect the assertions to be a bit more extensive, i.e. testing for the presence of the keys you would need in the PORO, testing the presence of totalHits, testing the presence of the `foods` array, etc